yellowbrick/text/postag.py

```diff
@@ -232,10 +230,14 @@ def _handle_universal(self, X,y):
                     for _, tag in tagged_sent:
                         if tag == "SPACE":
                             continue
-                        counter = self.pos_tag_counts_[y[idx]] if self.stack else self.pos_tag_counts_
+                        if self.stack:
+                            counter = self.pos_tag_counts_[y[idx]] 
+                        else: 
+                            counter = self.pos_tag_counts_['documents']
```

Comment:
no need to change, but consider the following as a stylistic thing you might like:

```python
label = y[idx] if self.stack else 'documents'
counter = self.pos_tag_counts[label]
```